ivy/functional/frontends/torch/tensor.py

```diff
@@ -1411,6 +1411,11 @@ def div_(self, other, *, rounding_mode=None):
         self.ivy_array = self.div(other, rounding_mode=rounding_mode).ivy_array
         return self
 
+    @with_supported_dtypes({"2.0.1 and below": ("float16", "float32", "float64", "bfloat16")}, "torch")
+    def true_divide_(self, other, *, rounding_mode=None):
+        self.ivy_array = torch_frontend.div(self, other).ivy_array
```

Comment:
@jieunboy0516, I have made the necessary modifications. Thank you for your guidance! 😊

Reply:
Hi, sorry for the confusion, I meant that `rounding_mode=None` should be explicitly passed to `self.div`! As according to the doc:
`Alias for [torch.div()](https://pytorch.org/docs/stable/generated/torch.div.html#torch.div) with rounding_mode=None.`
Thanks

Reply:
@jieunboy0516, I apologize for the confusion. I had mistakenly assumed that excluding the `rounding_mode` parameter from `self.div` would set it to `None` by default. I'm sorry for any misunderstanding this may have caused. 

Now, I have explicitly passed `rounding_mode=None` to `self.div`. Thank you very much. 🙂